doc/source/installing.rst

```diff
@@ -287,14 +287,18 @@ Clone the yt repository with:
 
   $ git clone https://github.com/yt-project/yt
 
-Once inside the yt directory, update to the appropriate branch and
-run ``pip install -e .``. For example, the following commands will allow you
-to see the tip of the development branch.
+Once inside the yt directory, update to the appropriate branch and run
+``pip install -e .``, or ``conda develop -b .``.  For example, the
+following commands will allow you to see the tip of the development
+branch.
 
 .. code-block:: bash
 
   $ git checkout master
+  $
   $ pip install -e .
+  $ # or
+  $ conda develop -b .
```

Comment:
Can we rephrase this so there's a separate code block for the conda instructions? People tend to skim these quickly so it's better in my opinion to err on the side of extra verbosity in the interest of clarity.

Reply:
sure thing.

Reply:
done.